### PR TITLE
accounts/usbwallet, common/bitutil: fix broken links in docs

### PR DESCRIPTION
**Pull request by justindhillon**
### Summery

There are multiple broken links in this project. Here is what I have fixed:

https://github.com/MikeMcl/bignumber.js/LICENCE --> https://github.com/MikeMcl/bignumber.js/blob/master/LICENCE.md

https://golang.org/src/crypto/cipher/xor_test.go --> https://go.dev/src/crypto/subtle/xor_test.go

https://golang.org/src/crypto/cipher/xor.go --> https://go.dev/src/crypto/subtle/xor.go

https://raw.githubusercontent.com/LedgerHQ/blue-app-eth/master/doc/ethapp.asc --> https://github.com/LedgerHQ/app-ethereum/blob/develop/doc/ethapp.adoc

### Support my work

These links were found with [link-inspector](https://github.com/justindhillon/link-inspector). If you find this PR useful, give the repo a ⭐

### CODE REVIEW

**Review by holiman — COMMENTED** [review]
Looks like I never submitted my comments that I wrote a few weeks ago :)

File: internal/jsre/deps/bignumber.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* bignumber.js v2.0.3 https://github.com/MikeMcl/bignumber.js/LICENCE */
```
[the remark this answers is not in the document]
Comment (justindhillon) [reply internal/jsre/deps/bignumber.js]: This has been reverted

File: common/bitutil/bitutil.go [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Adapted from: https://golang.org/src/crypto/cipher/xor.go
+// Adapted from: https://go.dev/src/crypto/subtle/xor.go
```
[the remark this answers is not in the document]
Comment (justindhillon) [reply common/bitutil/bitutil.go]: This has been changed

**Review by holiman — APPROVED** [review]
LGTM

**Pull request merged**